src/main/java/sirius/biz/process/logs/ProcessLog.java

```diff
@@ -52,11 +53,17 @@
 @Framework(Processes.FRAMEWORK_PROCESSES)
 public class ProcessLog extends SearchableEntity {
 
+    public static final int MESSAGE_TYPE_COUNT_ONE = 1;
```

Comment:
java doc (purpose) etc.

also - is 1 a feasible number?

Reply:
a single error could be a one-off problem, shouldn't always at least use 10?

Reply:
Added. In some imports we log the "certain situation exists" case which demanded a single logging. One must know exactly how to use it, but can drop if you think its superfluous.